packages/soy-core/test/setup.js

```diff
@@ -1,6 +1,11 @@
-const { SoyPublicResolver, ENS } = require('soy-contracts');
+const ENSArtifact = require('soy-contracts/build/contracts/ENSRegistry.json');
+const SoyPublicResolverArtifact = require('soy-contracts/build/contracts/SoyPublicResolver.json');
```

Comment:
Test cases need the whole build artifact for their binaries doing deploys below.